src/UALJs.test.ts

```diff
@@ -1,23 +1,37 @@
 import 'jest-localstorage-mock'
-import { User } from 'universal-authenticator-library'
+import promisePolyFill from 'promise'
+import { Authenticator, User } from 'universal-authenticator-library'
 import { UALJs } from './UALJs'
 
 import { AutologinAuthenticator } from './AuthMocks/AutologinAuthenticator'
 import { BaseMockAuthenticator } from './AuthMocks/BaseMockAuthenticator'
 
+jest.useFakeTimers()
+global.Promise = promisePolyFill
```

Comment:
As loathe as I am to suggests comments, I think one would be appropriate here, perhaps with a link to the Jest Github issue explaining why you had to use this.

Reply:
I had a link in the helper function `runPromises` at the end of the tests, but i agree i can add it here as well for clarity